src/scheduler/prime.c

```diff
@@ -467,16 +499,30 @@ load_day(enum days d, enum prime_time pr, char *tok)
 
 	if (tok != NULL) {
 		if (!strcmp(tok, "all") || !strcmp(tok, "ALL")) {
+			if (pr == NON_PRIME && conf.prime[d][PRIME].all == TRUE) {
+				/*
+				 * The user specified both all prime and all non-prime for this day
+				 * all prime wins in such a situation, so just ignore the non-prime info
+				 */
```

Comment:
Turn this comment into a more concise log message.  This is an error case, so we shouldn't just assume something without saying it.

Reply:
Just to be sure, you want me to actually log a message to sched logs here, or just trim the comment?

Reply:
The comment as a comment was fine.  I want a scheduler log message.  If we're adding a scheduler log message, I don't think we need both a comment and the log message.  For a log message, the comment is a bit wordy.